examples/CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
 add_subdirectory(boost_starter)
 
-add_subdirectory(catch2_existing_main)
-add_subdirectory(catch2_starter)
+add_subdirectory(catch2_v2_existing_main)
+add_subdirectory(catch2_v2_starter)
+
+add_subdirectory(catch2_v3_existing_main)
+add_subdirectory(catch2_v3_starter)
```

Comment:
supporting two catch2 versions implies that there need to be two times the example projects.